main.py

```diff
@@ -15,11 +13,23 @@
 def business_questions(spark, trip_fare_df, trip_data_df):
     """all business methods will be invoked here """
     query_manager = QueryManager(spark, trip_fare_df, trip_data_df)
-    # Which day of the week has the highest number of trips?
-    trips_fare_by_week = query_manager.trips_count(trip_fare_df, "pickup_datetime")
-    trips_data_by_week = query_manager.trips_count(trip_data_df, "pickup_datetime")
-    trips_fare_by_week.show(20)
-    trips_data_by_week.show(20)
+    ## Which day of the week has the highest number of trips?
+    trips_by_week = query_manager.trips_count(columns.pickup_datetime)
+    trips_by_week.show(20)
```

Comment:
We are supposed to write results of our questions in files (for example, using CSVManager). You can fix this here or we can fix it later when cleaning up the project.